ark/segmentation/marker_quantification.py

```diff
@@ -27,26 +144,32 @@ def compute_marker_counts(input_images, segmentation_labels, nuclear_counts=Fals
             boolean flag to determine whether nuclear counts are returned
         regionprops_features (list):
             morphology features for regionprops to extract for each cell
+        regionprops_extras (list):
+            list of extra properties derived from regionprops to compute
         split_large_nuclei (bool):
             controls whether nuclei which have portions outside of the cell will get relabeled
         extraction (str):
             extraction function used to compute marker counts.
         **kwargs:
-            arbitrary keyword arguments
+            arbitrary keyword arguments for get_cell_props
+
     Returns:
         xarray.DataArray:
             xarray containing segmented data of cells x markers
     """
 
     misc_utils.verify_in_list(
         extraction=extraction,
-        extraction_options=list(extraction_function.keys())
+        extraction_options=list(EXTRACTION_FUNCTION.keys())
     )
 
     if regionprops_features is None:
         regionprops_features = ['label', 'area', 'eccentricity', 'major_axis_length',
                                 'minor_axis_length', 'perimeter', 'centroid']
 
+    if regionprops_extras is None:
```

Comment:
I think we've been overusing this pattern. It makes are our function definitions less clear. If we're going to assign this as the default, just make this the default in the function. ```None``` makes sense as the default when we need to compute something, and hence can't set an informative default. However, if we can set the default, we should just set it.

Reply:
OK sounds good. This would make a perfect candidate for ark.settings.

Reply:
If they're going to be used in multiple functions, then I agree. If it's just going to be used for a single function, then probably overkill. I'm guessing some of these will be copied multiple places, others only once, so you'll be able to decide on a case-by-case basis